### PR TITLE
ZnTrack [0.1.5]

### LINKED ISSUE #95: Allow with `exec_=True` to run the stage silently / do not show stdout/stderr at all.
Simply add this as another option to the call decorator

### LINKED ISSUE #99: remove `PyYAML` from the dependencies
It seems like PyYAML is not used within ZnTrack anymore.

### LINKED ISSUE #105: ZnTrack requires numpy!

### LINKED ISSUE #108: Node depends on `dvc.outs` but `dvc.results` does not change
Currently there is no documentation about a Node that only depends on the `dvc.outs` and `dvc.results` does not change.
Something like:

```py
from zntrack import Node, dvc
@Node()
class Prepare:
    outs = dvc.outs('test.in')
    
    def run(self):
        pass

@Node()
class Test():
    prepare_node = dvc.deps(Prepare(load=True)) # this does not work!
    prepare = dvc.deps(Prepare(load=True).outs)
    
    def run(self):
        pass
```

### LINKED ISSUE #109: Document metric/plot
There is currently no documentation about using `dvc.metrics` or `dvc.plots` this would be very useful to add!

### PR DESCRIPTION
**Pull request by PythonFZ**
- fix #105 
- fix #108 
- fix #99 
- fix #95 
- fix #109

### CODE REVIEW

File: zntrack/dvc/__init__.py [hunk 1]
```diff
@@ -32,6 +32,7 @@ class params(ZnTrackOption):
 
 class result(ZnTrackOption):
```
Comment (SamTov) [inline zntrack/dvc/__init__.py:33]: class should be documented.

**Review by SamTov — APPROVED** [review]
The changes all look good but there is a class in there that should be documented.

File: zntrack/dvc/__init__.py (hunk 1 above)
Comment (PythonFZ) [reply zntrack/dvc/__init__.py:32]: Added description in 089cf8e

**Comment by PythonFZ** [comment]
> The changes all look good but there is a class in there that should be documented.
> 
> In general for the PR can you give a short overview of why you made the changes?

I added the docs to the class you mentioned.
Maybe I updated the description after your review, but this PR fixes the above mentioned issues.

**Pull request merged**